docker/Dockerfile

```diff
@@ -119,9 +120,9 @@ FROM ubuntu:22.04
 
 RUN apt-get update -y \
     && apt-get install -y build-essential \
+    libev-dev \
```

Comment:
oh no! libevent :-p

Reply:
One of the dependencies needed it - I didn't dig any further than that. If its a concern I can? :man_shrugging: